_posts/2018-03-18-phoenix-live-view.md

```diff
@@ -0,0 +1,341 @@
+---
+author: Sophie DeBenedetto
+author_link: https://github.com/sophiedebenedetto
+categories: til
+date: 2018-12-17
+layout: post
+title:  Walk-Through of Phoenix Live View
+excerpt: >
+  Learn how to use Phoenix Live View for real-time features without complicated JS frameworks.
+---
+It's here! Phoenix Live View leverages server-rendered HTML and Phoenix's native WebSocket tooling so you can build fancy real-time features without all that complicated JavaScript. If you're sick to death of writing JS (I had a bad day with Redux, don't ask), then this is the library for you!
+
+Phoenix Live View is brand brand new so I thought I'd provide short write-up of a super simple demo I built for anyone looking to get up and running.
+
+## What is Live View?
+
+Chris McCord said is best in his [announcement](https://dockyard.com/blog/2018/12/12/phoenix-liveview-interactive-real-time-apps-no-need-to-write-javascript) back in December:
+
+> Phoenix LiveView is an exciting new library which enables rich, real-time user experiences with server-rendered HTML. LiveView powered applications are stateful on the server with bidrectional communication via WebSockets, offering a vastly simplified programming model compared to JavaScript alternatives.
+
+## Kill Your JavaScript
+
+If you've waded through an overly complex SPA that Reduxes all the things (for example), you've felt the maintenance and iteration costs that often accompany all that fancy JavaScript.
+
+Phoenix Live View feels like the perfect fit for the 90% of the time that you do want some live updates but don't actually need the wrecking ball of many modern JS frameworks.
```

Comment:
There's a bunch of "the" in a row, what do you think of:  "feels like a perfect fit"